### PR TITLE
Write a guide for deploying Armada to AKS #1760

### PR DESCRIPTION
**Pull request by sarthaksarthak9**
#### 1760

┆Issue is synchronized with this [Jira Task](https://gr-oss.atlassian.net/browse/BATCH-117) by [Unito](https://www.unito.io)

### CODE REVIEW

File: GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Azure CLI installed and configured
+Helm installed
+Git installed
+## Step 1: Clone Armada Repository
+
+The first step is to clone the Armada repository to your local machine. You can do this by running the following command:
+
+#### bash
+
+git clone https://github.com/armadaplatform/armada.git
```
Comment (kannon92) [inline GUIDE.md:15]:
```suggestion
git clone https://github.com/armadaproject/armada.git
```

**Comment by kannon92** [comment]
Hey @sarthaksarthak9 thank you for this! I’d love to see this working in action sometime. We don’t have any azure experience in staff so review might be tough. Can you test your deployment to see if it’s working? Submitting jobs, watching events

**Comment by sarthaksarthak9** [comment]
@kannon92 , yah i will do it

**Comment by ShivangShandilya** [comment]
hey @sarthaksarthak9, I would like to recommend reformatting the documentation. Currently, it looks quite unpresentable.

**Comment by sarthaksarthak9** [comment]
@ShivangShandilya  , yah sure!!

**Pull request closed without merging**